atomic_reactor/source.py

```diff
@@ -57,6 +57,8 @@ def __init__(self, build_path):
         self.go = self.data.get('go') or {}
         self.image_build_method = meth = self.data.get('image_build_method')
         self.inherit = self.compose.get('inherit', False) if self.compose else False
+        if self.compose:
+            self.compose.pop('inherit', None)
```

Comment:
I'd like a comment here explaining why this is useful, i.e. it means that self.compose can be evaluated as a bool in order to decide whether any ODCS composes will be created.

Reply:
ok will add